code/AssetLib/glTF2/glTF2Asset.inl

```diff
@@ -980,18 +981,20 @@ void Accessor::ExtractData(T *&outData) {
     }
 
     const size_t maxSize = GetMaxByteSize();
-    if (count * stride > maxSize) {
-        throw DeadlyImportError("GLTF: count*stride ", (count * stride), " > maxSize ", maxSize, " in ", getContextForErrorMessages(id, name));
+    if (usedCount * stride > maxSize) {
```

Comment:
I just noticed that this error check does not make sense if we pass an index list